### PR TITLE
Real time text revealing and some css style

### PR DESCRIPTION
**Pull request by swiftyloves**

### CODE REVIEW

**Comment by qcl** [comment]
I want to go to sleep now, so I will review and test (?) it tomorrow, thanks your pull request :+1:
React looks so COOL!

File: index.html [hunk 1]
```diff
@@ -30,90 +32,133 @@
...
-        drawWords();
-    };
-
-    previewBtn.onclick = function(){
-        ga("send","event","MasterQuote","Preview",textContent.value);
-        gen();
-    };
 
     downloadBtn.onclick = function(){
         ga("send","event","MasterQuote","Download",textContent.value);
```
Comment (qcl) [inline index.html:83]: Variable `textContext` here is not defined.

File: index.html [hunk 2]
```diff
@@ -30,90 +32,133 @@
...
+            this.drawWords(context);
+        },
+
+        componentDidUpdate: function () {
+            var canvas = document.getElementById("canvas");
+            var context = canvas.getContext("2d");
+
+            this.drawMaster(context);
+            this.drawWords(context);
+            ga("send","event","MasterQuote","Preview",textContent.value);
```
Comment (qcl) [inline index.html:120]: `textContent` here is not defined, too.

File: index.html [hunk 3]
```diff
@@ -30,90 +32,133 @@
...
+        drawMaster: function(context) {
+            context.clearRect.apply(context, this.imgArea);
+            var img = document.getElementById("soongPoster");
+            context.drawImage.apply(context, [img].concat(this.imgArea));
+        },
+
+        drawWords: function (context) {
+            var textCenter = (this.textArea[0] + this.textArea[2])/2;
+            var textContent = document.getElementById("text");
+            var textContent = this.state.text;
```
Comment (qcl) [inline index.html:136]: Just curious, why assigning `textContent` twice with different value?

**Comment by qcl** [comment]
Download button cannot work normally, but I will merge it then fix it. Thanks :D

File: index.html (hunk 3 above)
Comment (swiftyloves) [inline index.html:136]: I forgot to remove....
Sorry for so many bugs : (

Comment (qcl) [inline index.html:136]: No mind :p

**Pull request merged**